contrib/download-frozen-image-v2.sh

```diff
@@ -76,6 +77,14 @@ fetch_blob() {
 		curl -fSL "${curlArgs[@]}" \
 			"$blobRedirect" \
 			-o "$targetFile"
+	else
+		echo
+		echo "loop didn't trigger"
+		echo curlHeaders are: $curlHeaders
```

Comment:
Can you retry this with `echo -n "$curlHeaders" | hexdump` or something so we can see the exact bytes we're comparing against?